.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,55 @@
+<!--
+    Hello! Thanks for submitting a pull request to MusicBrainz Server.
+    We appreciate your time and interest in helping our project!
+
+    Please use this template to help us review your change.
+
+    Depending on your change, some sections may be unneeded, just remove these.
+    For example, refactor pull requests usually don’t need the section “Action”.
```

Comment:
```suggestion
    Depending on your change, some sections may be unneeded; just remove these.
    For example, refactoring pull requests usually don’t need the section “Action”.
```

Reply:
Replaced with “small” in https://github.com/metabrainz/musicbrainz-server/pull/1345/commits/f71b4d6f566f860cb6013adbf704da5bfa1fde34 as “refactoring” could be easily confused with “rebasing”.